client/extensions/hello-dolly/hello-dolly-page.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import Main from 'components/main';
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import FAQ from 'components/faq';
+import FAQItem from 'components/faq/faq-item';
+import { getCurrentLyric } from './state/selectors';
+import lyrics from './state/lyrics';
+import { nextLyric } from './state/actions';
+
+class HelloDollyPage extends Component {
```

Comment:
sorry for coming in late here on this (or for missing it) but we strongly encourage a "stateless-functional component-first" approach that only switches into a `Component` once a need for the lifecycle methods or `state` has been established. neither requirement seems to have been met here so I'm wondering why this is extending `Component` instead of being a simpler render-function.